android/assets/jsons/Civ V - Vanilla/Ruins.json

```diff
@@ -0,0 +1,64 @@
+[
+    {
+        "name": "freeCulture",
+        "notification": "We have discovered cultural artifacts in the ruins! (+20 culture)",
+        "uniques": ["Gain [20] [Culture]"]
+    },
+    {
+        "name": "joinWorker",
+        "notification": "A worker has joined us!",
```

Comment:
These notifications should be changed to "A [Worker] has joined us!"
If the unit unique is configurable, the notification should be so too

Reply:
This and the Settler notification, obviously

Reply:
The reason I did it this way, is that the parameter in the unique is filled when the modmaker writes the json file, and so already knows what unit will be added. If it was unknown (for example, with the 50-100 gold), then a parameter would make sense, as the true value will only be determined at runtime and so the modmaker couldn't fill it in in the json file.

Reply:
I'm saying that from a translation standpoint, this would reduce 1 translation now, and would mean that modded effects are also translated
Otherwise you need to translate for each unit separately

Reply:
That is a good point, I'll add it in